spec/requests/waste_exemptions_engine/contact_address_lookup_forms_spec.rb

```diff
@@ -3,7 +3,7 @@
 require "rails_helper"
 
 module WasteExemptionsEngine
-  RSpec.describe "Contact Address Lookup Forms", type: :request, vcr: true do
```

Comment:
This tag here was never useful since VCR is not configured for tags to work yet, hence I have removed it

Reply:
Is it not? I have been using `bundle exec -t vcr` when regenerating the cassettes and it seems to work.

Reply:
Well, when the tag is used then we don't need to deal with the `use_cassettes` methods while we do instead. Let me rollback this change then and let's tackle it separately :)

Reply:
Cool :) Can we apply the `vcr` tag to the new w3c validation test as well?